pyproject.toml

```diff
@@ -8,6 +8,10 @@ authors = ["xal9wiii4ik <xal9wiii4ik@yandex.ru>"]
 python = "^3.8"
 requests = "^2.26.0"
 Flask = "^2.0.2"
+pre-commit = "^2.16.0"
+mypy = "^0.930"
+flake8 = "^4.0.1"
+autoflake = "^1.4"
```

Comment:
these are dev dependencies, should go to next section

Reply:
can you explain what does it mean

Reply:
I understand now)

Reply:
you can do `poetry install` with `--no-dev `flag. in this case `dev` dependencies (those used for development and are not needed for the code to execute in prod) are not installed. you'll use that flag when building your local image, so it contains only stuff you need in prod

Reply:
understand